### PR TITLE
New Optin Template Assets and Instantiation

### PR DESCRIPTION
**Pull request by ShoboySnr**
Instantiate optin templates assets

### CODE REVIEW

File: config.codekit3 [hunk 1]
```diff
@@ -552,6 +549,32 @@
       "q" : 100,
       "rq" : 75
     },
+    "\/src\/assets\/images\/optin-themes\/blogger\/blogger-lightbox.png" : {
+      "ft" : 32768,
+      "iS" : 23014,
+      "oA" : 0,
+      "oAP" : "\/src\/assets\/images\/optin-themes\/blogger\/blogger-lightbox.png",
```
Comment (w3guy) [inline config.codekit3:556]: Ensure you exclude your codekit config file from every PR @ShoboySnr

File: src/assets/js/mailoptin.min.js [hunk 2]
```diff
@@ -1,2 +1,3309 @@
...
\ No newline at end of file
+(function ($, require$$0, mailoptin_globals) {
```
Comment (w3guy) [inline src/assets/js/mailoptin.min.js:1]: Remove your minfied mailoptin.js too @ShoboySnr

File: config.codekit3 (hunk 1 above)
Comment (ShoboySnr) [reply config.codekit3:556]: Okay will do that

File: src/assets/js/mailoptin.min.js (hunk 2 above)
Comment (w3guy) [reply src/assets/js/mailoptin.min.js:1]: @ShoboySnr please cleanup the PR so i can review

Comment (ShoboySnr) [reply src/assets/js/mailoptin.min.js:1]: Okay I'll

**Comment by w3guy** [comment]
@ShoboySnr why are there two optin images? Please remove one

**Comment by w3guy** [comment]
@ShoboySnr I forgot to tell you, all themes you build are premium themes as a result, they should go into libsodium Git repo instead. Move this there too.

My apologies.

**Comment by ShoboySnr** [comment]
> @ShoboySnr I forgot to tell you, all themes you build are premium themes as a result, they should go into libsodium Git repo instead. Move this there too.
> 
> My apologies.

Okay, will do that

**Comment by ShoboySnr** [comment]
@w3guy I think I placed it under the premium_themes() function in this repository. Isn't that how it should be? I see other premium themes there

**Comment by w3guy** [comment]
Yoj also need to move the class to libsofium folder.

Premium optin themes are not found in Core.

Don’t worry, it will be easy

**Comment by ShoboySnr** [comment]
@w3guy I have moved it. So closing this PR.

**Comment by w3guy** [comment]
@ShoboySnr no ves, We need to reopen this o

**Comment by w3guy** [comment]
I thought the blogger theme file was also in Core.

No vex abeg

**Pull request closed without merging**